integration-tests/cloud-wallet.test.ts

```diff
@@ -0,0 +1,149 @@
+import {IWallet} from '@docknetwork/wallet-sdk-core/lib/types';
+import {closeWallet, createNewWallet} from './helpers/wallet-helpers';
+import {DataStore} from '@docknetwork/wallet-sdk-data-store/src/types';
+import {SYNC_MARKER_TYPE, initializeCloudWallet} from '@docknetwork/wallet-sdk-core/src/cloud-wallet';
+import {createDataStore} from '@docknetwork/wallet-sdk-data-store-typeorm/src';
+import {edvService} from '@docknetwork/wallet-sdk-wasm/src/services/edv';
+
+const EDV_URL = process.env.EDV_URL;
+const EDV_AUTH_KEY = process.env.EDV_AUTH_KEY;
+
+describe('Cloud wallet', () => {
+  let dataStore: DataStore;
+  let waitForEdvIdle: any;
+  let pushSyncMarker: any;
+  let findDocumentByContentId: any;
+  let updateDocumentByContentId: any;
+  let pullDocuments: any;
+  let getSyncMarkerDiff: any;
+  let clearEdvDocuments: any;
+  let wallet: IWallet;
+
+  beforeAll(async () => {
+    const {verificationKey, agreementKey, hmacKey} =
+      await edvService.generateKeys();
+
+    dataStore = await createDataStore({
+      databasePath: ':memory:',
+      dbType: 'sqlite',
+      defaultNetwork: 'testnet',
+    });
+
+    ({
+      waitForEdvIdle,
+      pushSyncMarker,
+      findDocumentByContentId,
+      updateDocumentByContentId,
+      pullDocuments,
+      getSyncMarkerDiff,
+      clearEdvDocuments,
+    } = await initializeCloudWallet({
+      dataStore,
+      edvUrl: EDV_URL,
+      agreementKey,
+      verificationKey,
+      hmacKey,
+      authKey: EDV_AUTH_KEY,
+    }));
+
+    await clearEdvDocuments();
+
+    wallet = await createNewWallet({
+      dontWaitForNetwork: true,
+      dataStore,
+    });
+  });
+
+  it('should see a document added directly to the EDV appear in the wallet after pulling', async () => {
+    const newDocId = `${Date.now()}`;
+    const newDoc = {
+      id: newDocId,
+      type: 'test',
+      data: 'test',
+    };
+
+    await edvService.insert({
+      document: {
+        content: newDoc,
+      },
+    });
+
+    await pullDocuments();
+
+    const sdkDocument = await dataStore.documents.getDocumentById(newDocId);
+
+    expect(sdkDocument).toEqual(newDoc);
+  });
+
+  // it('should sync a document added to the wallet to the EDV', async () => {
```

Comment:
any reason all below here is commented?

Reply:
it was a mistake, I've already uncommented that in a separate PR